HHS/Head-Start-TTADP
# [TTAHUB-3409] Trim title on RTR objective creation

## Description of change

There's one spot in our app where we weren't trimming objective titles on creation. This can lead to duplicate objectives (see support case)

This PR closes off that one case

## How to test
Create a goal and then an objective on that goal via the RTR with a space at the end of the objective
Select that goal and objective on an AR and save the report
Confirm that only one objective exists and no duplicates were created

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3409


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
